packages/core/core-flows/src/order/workflows/create-complete-return.ts

```diff
@@ -365,10 +365,6 @@ export const createReturnOrderWorkflow = createWorkflow(
         shipping_method: shippingMethodData,
         created_by: input.created_by,
       }),
-      updateOrderTaxLinesStep({
-        order_id: input.order_id,
-        shipping_methods: [shippingMethodData as any], // The types does not seems correct in that step and expect too many things compared to the actual needs
-      }),
```

Comment:
q: is this being removed on purpose?

Reply:
Yes, we shouldn't recalculate taxes for existing line items here.